instrumentation/opentelemetry_oban/test/opentelemetry_oban/plugin_handler_test.exs

```diff
@@ -96,14 +96,14 @@ defmodule OpentelemetryOban.PluginHandlerTest do
 
     assert_receive {:span,
                     span(
-                      name: "Elixir.Oban.Plugins.Stager process",
+                      name: "oban.plugin Oban.Plugins.Stager",
                       events: events,
                       status: ^expected_status
                     )}
 
     [
       event(
-        name: "exception",
+        name: :exception,
```

Comment:
This was already fixed separately in https://github.com/open-telemetry/opentelemetry-erlang-contrib/pull/558